src/_includes/docs/swift-package-manager/migrate-swift-plugin.md

```diff
@@ -90,37 +93,34 @@ The example below uses `ios`, replace `ios` with `macos`/`darwin` as applicable.
 
    ```swift title="Package.swift"
    let package = Package(
+       // TODO: Update your plugin name.
        name: [!"plugin_name"!],
        platforms: [
-           // The platforms your plugin supports.
-           // If your plugin only supports iOS, remove `.macOS(...)`.
-           // If your plugin only supports macOS, remove `.iOS(...)`.
            .iOS("12.0"),
            .macOS("10.14")
        ],
        products: [
+           // TODO: Update your library and target names.
            // If the plugin name contains "_", replace with "-" for the library name
            .library(name: [!"plugin-name"!], targets: [[!"plugin_name"!]])
        ],
        dependencies: [],
        targets: [
            .target(
+               // TODO: Update your target name.
                name: [!"plugin_name"!],
                dependencies: [],
                resources: [
-                   // If your plugin requires a privacy manifest, for example if it uses
-                   // any required reason APIs, update the PrivacyInfo.xcprivacy file to
-                   // describe your plugin's privacy impact, and then uncomment these lines.
+                   // TODO: If your plugin requires a privacy manifest
+                   // (e.g. if it uses any required reason APIs), update the PrivacyInfo.xcprivacy file
+                   // to describe your plugin's privacy impact, and then uncomment this line.
                    // For more information, see:
                    // https://developer.apple.com/documentation/bundleresources/privacy_manifest_files
                    // .process("PrivacyInfo.xcprivacy"),
    
-                   // If you have other resources that need to be bundled with your plugin, refer to
+                   // TODO: If you have other resources that need to be bundled with your plugin, refer to
                    // the following instructions to add them:
                    // https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package
-               ],
-               cSettings: [
-                   .headerSearchPath("include/[!plugin_name!]")
                ]
```

Comment:
This was a copy-paste mistake. The `cSettings` should only be used for Objective-C plugins.